scripts/basic-workflow-tests.sh

```diff
@@ -162,6 +167,23 @@ test_convert() {
     ilab model convert
 }
 
+test_config_value() {
+    task "Check configuration value '$1' in property '$3' of section '$2' of '$4'"
+
+    VALUE=$1
+    SECTION=$2
+    PROPERTY=$3
+    CONFIG_FILE=$4
+
+    PROPERTY_VALUE=$(grep -A50 "$SECTION"':' "$CONFIG_FILE" | grep "$PROPERTY"':' | xargs)
```

Comment:
How about actually parsing the yaml? That seems like it'd be a bit more robust. `yq` would do the trick.

Reply:
I thought about using `yq` alright but I didn't want to add another tool.  If people think its a better implementation, I can do that.

Reply:
`yq` is part of the runner's Ubuntu image https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md#tools. `scripts/functional-tests.sh` uses that image.

Reply:
Ok, now using `yq` instead.